jsonschema-validation.xml

```diff
@@ -514,18 +514,12 @@
             <section title="Foreword">
                 <t>
                     Structural validation alone may be insufficient to allow an application to correctly
-                    utilize certain values.  The "format" annotation keyword is defined to allow schema
+                    utilize certain values. The "format" annotation keyword is defined to allow schema
```

Comment:
can we keep unrelated whitespace changes out of the PR? it adds confusion and often conflicts with other things in flight.

Reply:
This was requested by @Relequestual in Slack.